src/ddmd/aggregate.d

```diff
@@ -21,6 +21,7 @@ import ddmd.declaration;
 import ddmd.dscope;
 import ddmd.dstruct;
 import ddmd.dsymbol;
+import ddmd.dsymbolsem;
```

Comment:
You could have used a selective import here.

Reply:
1. there'd not be much value in that, as dsymbolsem has only a handful of publics in it.

2. adding them now would bloat up these PRs by hundreds of lines.

3. I'm not real sure it is the right approach for dmd

Reply:
I think that while we should aim to reduce toplevel imports. Selective imports should probably be kept function local.

Eg: genCmain is the only user of the module ddmd.mars. Therefore it makes sense to remove the toplevel import of mars and encapsulate it into genCmain.